saleor/plugins/sendgrid/plugin.py

```diff
@@ -0,0 +1,255 @@
+            "help_text": (
+                "The dynamic template ID for account change email confirmation."
+            ),
+            "label": "Template ID for account change email confirm",
+        },
+        "account_change_email_request_template_id": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "The dynamic template ID for account change email request.",
+            "label": "Template ID for account change email request",
+        },
+        "account_password_reset_template_id": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "The dynamic template ID for account password reset.",
+            "label": "Template ID for account password reset",
+        },
+        "invoice_ready_template_id": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "The dynamic template ID for invoice ready.",
+            "label": "Template ID for invoice ready",
+        },
+        "order_confirmation_template_id": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "The dynamic template ID for order confirmation.",
+            "label": "Template ID for order confirmation",
+        },
+        "order_confirmed_template_id": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "The dynamic template ID for order confirmed.",
+            "label": "Template ID for order confirmed",
+        },
+        "order_fulfillment_confirmation_template_id": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "The dynamic template ID for order fulfillment confirmation.",
+            "label": "Template ID for order fulfillment confirmation",
+        },
+        "order_fulfillment_update_template_id": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "The dynamic template ID for order fulfillment update",
+            "label": "Template ID for order fulfillment update",
+        },
+        "order_payment_confirmation_template_id": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "The dynamic template ID for order payment confirmation.",
+            "label": "Template ID for order payment confirmation",
+        },
+        "order_canceled_template_id": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "The dynamic template ID for order canceled",
+            "label": "Template ID for order canceled",
+        },
+        "order_refund_confirmation_template_id": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "The dynamic template ID for order refund confirmation",
+            "label": "Template ID for order refund confirmation",
+        },
+        "api_key": {
+            "type": ConfigurationTypeField.SECRET,
+            "help_text": (
+                "This allows your application to authenticate to our API and send mail."
+            ),
```

Comment:
It allows our application (Saleor) to authenticate to Sendgrid API and send an email. But I think it would be enough to say: 

```suggestion
            "help_text": ("Your Sendgrid API key."),
```